### ISSUE TITLE
FB falied to authenticate error not showing up

### ISSUE THREAD

**Issue by hearsid** [tinderjs/tinderjs#34]
When I run the bot.js file from the terminal I don't see any error telling me that the FB authentication has failed instead it shows me some other error . 
I tried the same with the original tinderjs library and it is working there , please check the attached screenshots of both this and the original library 
<img width="963" alt="screen shot 2016-04-24 at 2 27 02 pm" src="https://cloud.githubusercontent.com/assets/3892451/14766467/c5d91f68-0a28-11e6-94ea-8c386dc28e51.png">
<img width="1033" alt="screen shot 2016-04-24 at 2 27 25 pm" src="https://cloud.githubusercontent.com/assets/3892451/14766468/c5e0d924-0a28-11e6-9a66-521019f9e496.png">

.

**Comment by run1t** [comment]
Can you try just with `data` in place of `data.results` ?
I think we have just forgot to modify the readme, if you confirm me that i will do a PR.

**Comment by run1t** [comment]
My bad I read your issue too fast...
by the way it's weird, my output with a wrong token is that 
```
{ status: 401,
  error: 'MissingAuthToken›missing x-auth-token header' }
{ status: 401,
  error: 'MissingAuthToken›missing x-auth-token header' } 
```
I will investigate

**Comment by jasongornall** [comment]
bump this is a deal breaker for using your library

**Comment by mrmayfield** [comment]
Yup this was changed a while back to the less than desirable functionality but I think a reversion is in order.
